compliance/collection/main.go

```diff
@@ -37,6 +41,31 @@ var (
 	once sync.Once
 )
 
+// These options control behaviour of all node inventory related functions
+type cachedScanOpts struct {
+	nodeName            string
+	scanner             nodeinventorizer.NodeInventorizer
+	inventoryCachePath  string
+	backoffWaitCallback func(int64) int64
+}
+
+// waitAndIncreaseBackoff returns the duration the backoff should wait next time
+func waitAndIncreaseBackoff(waitTimeSeconds int64) int64 {
+	maxBackoffSeconds := int64(env.NodeInventoryMaxBackoff.DurationSetting() / time.Second)
+	backoffIncrementSeconds := int64(env.NodeInventoryBackoffIncrement.DurationSetting() / time.Second)
+
+	if waitTimeSeconds > maxBackoffSeconds {
+		log.Warnf("Backoff interval hit upper boundary. Cutting from %d to %d", waitTimeSeconds, maxBackoffSeconds)
```

Comment:
That's not something users need to be concerned about. I suggest to decrease severity from Warn to Debug, if you still want to keep this log.

Reply:
Done!